forta-scanners-under-capacity/src/agent.js

```diff
@@ -159,8 +135,9 @@ function provideHandleTransaction(
         const scannerChainIdNormalized = scannerChainId.toNumber();
         scannersCountByChainId[scannerChainIdNormalized]++;
         const tokenIdString = tokenId.toString();
+        scannerIds.push(ethers.BigNumber.from(tokenIdString).toString());
```

Comment:
Can't we just push the `tokenIdString`? Why do we convert it to BigNumber and then back to string?

Reply:
When we convert it to bignumber we get the unique uint256 value of the scannerID which is required by the contracts to get data from them, and then I convert it to string so that I can use the ID later on when checking scanner capacity